armi/cases/tests/test_cases.py

```diff
@@ -262,7 +262,12 @@ def test_clone(self):
 
 
 class TestCaseSuiteDependencies(unittest.TestCase):
-    """CaseSuite tests."""
+    """CaseSuite tests.
+
+    .. test:: Dependence allows for one case to start after the completion of another.
+        :id: T_ARMI_CASE_SUITE0
+        :tests: R_ARMI_CASE_SUITE
```

Comment:
Not suggesting this happens right now, but do you think there will be a future effort to tie these sorts of tests breadcrumbs to individual unit tests rather than the entire test class?